iqama_fetcher.py

```diff
@@ -71,16 +88,31 @@ def FormatTime(time):
 
   # if the time input is in a format where it contains integers
   # then proceed forward otherwise return None
-  if re.match(r'(\d+)', time, flags=0):
+  if re.match(r'(\d+)', time):
     # strip the time variable from anything added to it
     # example: 10:00 PM becomes 1000 and 5 AM becomes 5
     time = ''.join(re.findall(r'(\d+)', time))
     # if time length is more than 2 characters then add a : after
     # the second character from the right
     # example: 1000 becomes 10:00 and 5 stays 5
     if len(time) > 2:
-      return time[:1] + ':' + time[1:] 
-    return time + ':00' 
+      return time[:1] + ':' + time[1:]
+    return time + ':00'
+
+
+# Fajr prayer matches an 'f' at the beginning and an 'r' at the end
```

Comment:
you can move these to the top of the file - where we typically place module-global constants